File: source/repos/BlackJack/BlackJackViewModels/GameViews/PlayGameView.cs
using BlackJack.DataAccess.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace BlackJack.ViewModels.GameViews
{
    public class PlayGameView
    {
        [Required(ErrorMessage = "PlayerId is required")]
        public Guid PlayerId { get; set; }

        [Required(ErrorMessage = "Numbers of bots is required")]
        [Range(1, 5,ErrorMessage ="Range from 1 to 5")]
        public int NumberOfBots { get; set; }

        public string Status { get; set; }
        public string Winner { get; set; }
        public string PlayerName { get; set; }
        public List<PlayGameCardsViewItem> PlayerCards { get; set; }
        public List<PlayGameBotsViewItem> Bots { get; set; }

        public PlayGameView()
        {
            PlayerCards = new List<PlayGameCardsViewItem>();
            Bots = new List<PlayGameBotsViewItem>();
        }
    }

    public class PlayGameBotsViewItem
    {
        public string BotName { get; set; }
        public List<PlayGameCardsViewItem> BotCards { get; set; }
    }
    public class PlayGameCardsViewItem
    {
        public CardRank StepRank { get; set; }
        public CardSuit StepSuit { get; set; }
    }
}

File: source/repos/BlackJack/BlackJack/ClientApp/src/app/shared/entities/play-game.view.ts
export class PlayGame {
    playerId: string;
    gameId: string;
    numberOfBots: number;
    status: string;
    winner: string;
    playerName: string;
    playerCards:[PlayGameCardsViewItem];
    bots:[PlayGameBotsViewItem];
}
export class PlayGameBotsViewItem{
    botName:string;
    botCards:[PlayGameCardsViewItem];
}
export class PlayGameCardsViewItem{
    stepRank:number;
    stepSuit:number;
}